rental.rb

```diff
@@ -1,7 +1,7 @@
 class Rental
   attr_reader :person, :book, :date
 
-  def initialize(date, person, book)
+  def initialize(date, book, person)
```

Comment:
- [ ] [OPTIONAL] When the user enters a non-existent id for a person when creating rentals, the app breaks, kindly consider using a condition to prevent the user from providing a non-existent id when creating the rentals

![Screenshot_20220804_073824](https://user-images.githubusercontent.com/25227022/182780342-9c522ce7-38ab-44bc-8735-2c54a264d3ca.png)